topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -690,7 +699,7 @@ For both within or between-sample comparison, the gene counts need to be normali
 
 This expression analysis is estimated from read counts and attempts are made to correct for variability in measurements using replicates, that are absolutely essential for accurate results. For your own analysis, we advise you to use at least 3, but preferably 5 biological replicates per condition. It is possible to have different number of replicates per condition.
```

Comment:
Does it make sense to explain briefly the difference between biological and technical replicates or at least to make them aware that there is a difference

Reply:
Yep we should add an explanation there!